File: openapi/code_samples/go/api_v1_users_{id}/patch.go
upd := &usersv1.UpdateUser{
    UserProfile: &usersv1.UpdateUserProfile{
        FirstName: "John",
        LastName:  "Smith",
    },
    Metadata: map[string]string{
        "department": "sales",
    },
}
scalekitClient.User().UpdateUser(ctx, "usr_123", upd)

File: openapi/code_samples/python/api_v1_users_{id}/patch.py
from scalekit.v1.users.users_pb2 import UpdateUser
from scalekit.v1.commons.commons_pb2 import UserProfile

update_user = UpdateUser(
    user_profile=UserProfile(
        first_name="John",
        last_name="Smith",
    ),
    metadata={"department": "sales"},
)
response = scalekit_client.users.update_user("usr_123", update_user)
